app/views/users/new.html.erb

```diff
@@ -0,0 +1,47 @@
+<!--
+  <form accept-charset="UTF-8"  action="/users" method="post">
+    <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+    <label for="user[username]">Username</label>
+    <input type="text" name="user[username]" id="user[username]"><br>
+
+    <label for="user[email]">Email</label>
+    <input type="email" name="user[email]" id="user[email]"><br>
+
+    <label for="user[password]">Password</label>
+    <input type="password" name="user[password]" id="user[password]"><br>
+
+    <input type="submit" value="Submit">
+  </form> 
+-->
+
+
+<%- form_tag('/users', method: 'post') do -%>
+
+  <%- label_tag(:username, 'Username') -%>
+  <%- text_field_tag(:username) -%>
+
+  <%- label_tag(:email, 'Email') -%>
+  <%- email_field_tag(:email) -%>
+
+  <%- label_tag(:password, 'Password') -%>
+  <%- password_field_tag(:password) -%>
+
+  <%- submit_tag("Search") -%>
+<%- end -%>
```

Comment:
- [ ] form_tag should be commented out too, we only keep the form_for :+1:

Reply:
Hello @samgaco, thank you for your feedback. If you open the edit.html.erb/new.html.erb files in your browser, you will see that they have actually been commented out.

Best Regards